Core/Core/Assignments/AssignmentList/ViewModel/AssignmentCellViewModel.swift

```diff
@@ -62,15 +84,11 @@ public class AssignmentCellViewModel: ObservableObject {
             return String(localized: "Multiple Due Dates", bundle: .core)
         }
 
-        if let dueAt = assignment.dueAt {
-            let format = String(localized: "Due %@", bundle: .core, comment: "i.e. Due <Jan 10, 2020 at 9:00 PM>")
-            return String.localizedStringWithFormat(format, dueAt.relativeDateTimeString)
+        if let dueAt = formattedDueAt {
+            let format = String(localized: "Due %@", bundle: .core, comment: "i.e. Due <Jan 10, 2020 9:00 PM>")
+            return String.localizedStringWithFormat(format, dueAt)
         }
 
         return String(localized: "No Due Date", bundle: .core)
```

Comment:
There is an `assignment.dueText` (from protocol `DueViewable`), which could be used here.
This is the one used on Grades screen as well.
It shouldn't matter what example is in figma, the date/time should always be localized.
(the `formattedDueAt` defined above is not localized. it could be removed after this change)

Please also make sure, when you test this use something like this, not a string literal, because that can fail on machines with different localization.
```
XCTAssertEqual(dueText, "Due " + TestConstants.date1225.relativeDateTimeString)
```